packages/server/graphql/resolvers.ts

```diff
@@ -145,3 +147,37 @@ export const resolveForBillingLeaders = (fieldName) => async (
   const isBillingLeader = await isUserBillingLeader(viewerId, orgId, dataLoader)
   return isBillingLeader || isSuperUser(authToken) ? source[fieldName] : undefined
 }
+
+export const resolveJiraIssue = async (source, _args, {authToken, dataLoader}) => {
+  const {teamId, serviceTaskId} = source
+  const [cloudId, issueKey] = serviceTaskId
+    ? serviceTaskId.split(':')
+    : [source.cloudId, source.key]
+  const viewerId = getUserId(authToken)
+  // we need the access token of a person on this team
+  const teamAuths = await dataLoader.get('atlassianAuthByTeamId').load(teamId)
+  const [teamAuth] = teamAuths
+  if (!teamAuth) {
+    sendToSentry(new Error('No atlassian access token exists for team'), {userId: viewerId})
+    return null
+  }
+  const {userId} = teamAuth
+  const accessToken = await dataLoader.get('freshAtlassianAccessToken').load({teamId, userId})
+  const manager = new AtlassianServerManager(accessToken)
+  const issueRes = await manager.getIssue(cloudId, issueKey)
+  if ('message' in issueRes) {
+    sendToSentry(new Error(issueRes.message), {userId: viewerId})
+    return null
```

Comment:
returning null isn't great because if the client hits an error, they should see what the error is.